app/packages/dataset/src/Dataset.tsx

```diff
@@ -59,6 +59,7 @@ export interface DatasetProps {
   readOnly?: boolean;
   theme?: "dark" | "light";
   toggleHeaders?: () => void;
+  canEditSavedViews?: boolean;
```

Comment:
this file is not used but I am keeping it up to date with the one in teams in case